vulnerabilities/api.py

```diff
@@ -71,36 +73,64 @@ class MinimalVulnerabilitySerializer(serializers.HyperlinkedModelSerializer):
 
     class Meta:
         model = Vulnerability
-        fields = ["url", "vulnerability_id", "references", "summary"]
+        fields = ["url", "vulcoid", "summary", "references"]
```

Comment:
IMHO the field should be called still `vulnerability_id` and it should always be prefixed by "VULCOID"